src/subdomains/supporting/payment/services/transaction-helper.ts

```diff
@@ -173,7 +173,7 @@ export class TransactionHelper implements OnModuleInit {
     to: Active,
     paymentMethodIn: PaymentMethod,
     paymentMethodOut: PaymentMethod,
-    allowExpiredPrice: boolean,
+    allowCachedBlockchainFee: boolean,
```

Comment:
Here we still want to select, if the price can be expired or not. For payment info calls, we always use cached blockchain fees.